ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -182,16 +182,18 @@ export default function CreatePassword({
             titleText={t('newPassword')}
             value={password}
             titleDetail={
-              <button
-                className="create-password__form--password-button"
-                type="button"
-                onClick={(e) => {
-                  e.preventDefault();
-                  setShowPassword(!showPassword);
-                }}
-              >
-                {showPassword ? t('hide') : t('show')}
-              </button>
+              <Typography variant={TYPOGRAPHY.H7}>
+                <a
```

Comment:
I loathe using an `<a>` instead of `<button>` but it appears `<button>` elements inside of a `<label>` cause some really gross behavior which led to this issue.

Reply:
only thing is that now the `hide`/`show` button is showing up as blue instead of black. Maybe we can style it back?

Reply:
I did that on purpose, as we should users it's a clickable element.  Before it it bled in, which looked weird.

Reply:
Is using anchor tag here not against semantic use of html tags ?

Reply:
Do you have an alternative?  `<button>` is causing the problem.